client/state/index.js

```diff
@@ -80,6 +81,7 @@ const reducers = {
 	currentUser,
 	documentHead,
 	domains,
+	dropZone,
```

Comment:
Good call, moved the reducers to `state/ui/reducer`, since we only care for the global state of a `DropZone` being shown for now.